tests/helpers/reya_tester/data.py

```
@@ -32,37 +31,52 @@ def __init__(self, tester: "ReyaTester"):
         last_exc: Optional[Exception] = None
         for attempt in range(max_attempts):
             try:
-                price_info: Price = await self._t.client.markets.get_price(symbol)
-                logger.info(f"Price info: {price_info}")
-                current_price = price_info.oracle_price
-                if current_price:
-                    logger.info(f"💰 Current market price for {symbol}: ${float(current_price):.2f}")
-                    return current_price
-                logger.info(f"❌ Current market price for {symbol} missing oracle_price")
+                current_price = (await self._t.client.get_asset_oracle_price(asset)).oracle_price
+                logger.info(f"💰 Current asset oracle price for {asset}: ${float(current_price):.2f}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)